### PR TITLE
Version increment 2.2.1

### PR DESCRIPTION
**Pull request by prudhvigodithi**
Signed-off-by: prudhvigodithi <pgodithi@amazon.com>

### Description
Version increment 2.2.1

### Issues Resolved
Part of: https://github.com/opensearch-project/opensearch-build/issues/2481

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by cliu123** [comment]
@prudhvigodithi Could you upgrade security plugin and add it to manifest first? The integration tests won't pass until then.

**Review by DarshitChanpura — APPROVED** [review]

File: opensearch_dashboards.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "id": "securityDashboards",
-  "version": "2.2.0.0",
+  "version": "2.2.1.0",
   "opensearchDashboardsVersion": "2.2.0",
```
Comment (DarshitChanpura) [inline opensearch_dashboards.json:4]: Can you please update this to 2.2.1.

**Review by davidlago — APPROVED** [review]

**Comment by cliu123** [comment]
> @prudhvigodithi Could you upgrade security plugin and add it to manifest first? The integration tests won't pass until then.

@prudhvigodithi Following up on this comment. [Security plugin 2.2 branch](https://github.com/opensearch-project/security/blob/2%2C2/build.gradle#L14) has not been bumped to 2.2.1 version, so the integration tests failed on this PR. Please bump the version of security plugin, and re-run integration tests on this PR.

**Comment by prudhvigodithi** [comment]
Hey @cliu123 security is already on 2.2.1 https://github.com/opensearch-project/security/blob/2.2/build.gradle#L16. The link you have shows wrong branch
Thank you

File: opensearch_dashboards.json (hunk 1 above)
Comment (prudhvigodithi) [reply opensearch_dashboards.json]: Just updated this @DarshitChanpura.

**Pull request merged**